cdap-ui/app/cdap/components/EntityListView/Welcome/index.tsx

```diff
@@ -76,7 +79,9 @@ export default class Welcome extends React.PureComponent<void, IWelcomeState> {
 
   private startTour = () => {
     this.close();
-    Tour.start();
+    (document.querySelector('[data-cy="navbar-hamburger-icon"]') as HTMLElement).click();
```

Comment:
spoke with @ajainarayanan about removing these explicit click events since the Drawer component already have an  `open` prop